content/posts/eks-vs-ecs.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Amazon ECS vs EKS for running CloudBees Core/Jenkins
+authors:
+  - "Logan Donley"
+date: 2020-01-13T05:05:15-04:00
+showDate: true
+tags: ["Kubernetes","aws","EKS","ECS"]
+draft: true
+---
+
+A question my colleagues and I are often asked is "if I'm already using ECS is there any point in moving to EKS (or Kubernetes in general) for running CloudBees Core?" Both Amazon ECS and EKS are great products, but we have a strong preference towards Kubernetes. We'll take a look at why that is in this post.
```

Comment:
Instead of "We'll take a look at why that is in this post." Perhaps just "We'll take a look at why in this post." Less wordy is usually better.